openreview/venue_request/venue_request.py

```diff
@@ -330,16 +330,21 @@ def setup_meta_review_stage(self):
     def setup_submission_revision_stage(self):
 
         submission_revision_stage_content = {
+            'submission_revision_name': {
+                'description': 'What should be the name of the submission revision button (e.g. Revision, Supplementary Material, Post-Decision Revision)? Leave empty for default option - Revision',
+                'value-regex': '.*',
```

Comment:
can you add `default: 'Revision`, I will pre-populate the note editor with the 'Revision' string.